.github/workflows/tests.yml

```diff
@@ -95,6 +95,11 @@ jobs:
             python-version: '3.12'
             # https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-2346
             pyside6-ver: '!=6.5.1'
+          # Temporary CI run using pre-release IPython
+          - name-suffix: 'Pre-release IPython'
+            os: ubuntu-22.04
+            python-version: '3.12'
+            extra-requirements: '-r requirements/testing/extra.txt'
```

Comment:
I will remove this particular CI run now as IPython 8.24 was released a few hours ago and all CI runs here will now be using the latest code. So yes, it would be good to have a run with IPython pinned. I would rather not include it in the minimum version job, so assuming we do not want to add a new CI run just for this then I will try adding the pin to one of the ubuntu `extra-requirements` runs.

Reply:
We've been lucky here. IPython 8.24 is only available for Python >= 3.10 so the ubuntu 3.9 CI run automatically tests for backwards compatibility. I have made it explicit in CI anyway, in case we want to move it to Python 3.10 when we drop support for 3.9.